Makefile

```diff
@@ -120,6 +120,7 @@ build-api:
 
 build-web:
 	@echo "++\n***** Building Web for AWS\n++"
+	@yarn workspace @ehpr/web build
```

Comment:
Export already runs build doesn't it?

Reply:
It was crashing because not able to locate build dir

Reply:
It is due to build common

Reply:
yarn workspace @ehpr/web build => a. build dependencies + b. web

Reply:
🤔 Can we remove the extra build from the export step then? This looks like it'll build web twice

Reply:
`yarn workspace @ehpr/web build` only builds dependencies for web and web. That a improvement over build-common.